### PR TITLE
FightPad! macropad

### PR DESCRIPTION
**Pull request by AlisGraveNil**
## Description

This is a 12-key macropad for fighting games

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [X] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [X] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [X] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/clawsome/fightpad/config.h [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x7767
+#define PRODUCT_ID      0x0000
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    AlisGraveNil
+#define PRODUCT         FightPad
+#define DESCRIPTION     QMK Firmware: FightPad
+
```
Comment (zvecr) [inline keyboards/clawsome/fightpad/config.h:29]:
```suggestion

```

File: keyboards/clawsome/fightpad/rules.mk [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
```
Comment (zvecr) [inline keyboards/clawsome/fightpad/rules.mk:25]:
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/clawsome/fightpad/config.h [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x7767
+#define PRODUCT_ID      0x0000
```
Comment (noroadsleft) [inline keyboards/clawsome/fightpad/config.h:24]:
```suggestion
#define PRODUCT_ID      0x481C
```

File: keyboards/clawsome/fightpad/info.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
+    "keyboard_name": "FightPad",
+    "url": "www.clawboards.xyz",
+    "maintainer": "AAClawson (AlisGraveNil)",
+    "width": 8,
+    "height": 2,
+    "layouts": {
+        "LAYOUT": {
+            "key_count": 12,
```
Comment (noroadsleft) [inline keyboards/clawsome/fightpad/info.json:9]:
```suggestion
```

File: keyboards/clawsome/fightpad/info.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+                {"label":"K11 (B6,E6)", "x":1, "y":1},
+                {"label":"K12 (B6,B4)", "x":2, "y":1},
+                {"label":"K13 (B6,B2)", "x":4, "y":1},
+                {"label":"K14 (B6,B3)", "x":5, "y":1},
+                {"label":"K15 (B6,B1)", "x":6, "y":1},
+                {"label":"K16 (B6,F7)", "x":7, "y":1}
+            ]
+        }
+    }
+    ,"meta": "https://noroadsleft.github.io/kbf_qmk_converter/"
```
Comment (noroadsleft) [inline keyboards/clawsome/fightpad/info.json:26]:
```suggestion
```

File: keyboards/clawsome/fightpad/readme.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# FightPad
+
+This is a 2x7 macropad.
+
+* Keyboard Maintainer: [AAClawson](https://github.com/AlisGraveNil)
+* Hardware Supported: Los Numeros, Pro Micro, Elite-C
```
Comment (noroadsleft) [inline keyboards/clawsome/fightpad/readme.md:6]:
```suggestion
* Hardware Supported: FightPad; Pro Micro or Elite-C
```

File: keyboards/clawsome/fightpad/readme.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+* Hardware Availability: www.clawboards.xyz
```
Comment (noroadsleft) [inline keyboards/clawsome/fightpad/readme.md:7]:
```suggestion
* Hardware Availability: [Clawsome Boards](https://www.clawboards.xyz/)
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
GitHub tip: You can apply multiple suggestions to a single commit by using the [Files Changed tab](https://github.com/qmk/qmk_firmware/pull/10507/files).

**Review by noroadsleft — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**